src/main/java/dekk/pw/pokemate/PokeMate.java

```diff
@@ -80,6 +75,16 @@ public PokeMate() throws IOException, LoginFailedException, RemoteServerExceptio
     }
 
     public static void main(String[] args) throws RemoteServerException, IOException, LoginFailedException {
+        if (args[0].length() == 0){
```

Comment:
If null make it use the default location that was set before instead of throwing an error.